src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -135,12 +133,26 @@ export class AccessibleBufferWidget extends DisposableStore {
 			this._accessibleBuffer.classList.add(CssClass.Active);
 			this._xtermElement.classList.add(CssClass.Hide);
 		}));
+	}
+
+	private _initializeTerminalListeners(): void {
 		this.add(this._instance.onDidRequestFocus(() => {
 			if (this._isActive()) {
-				this._bufferEditor.focus();
+				this._editorWidget.focus();
+			}
+		}));
+		this.add(Event.runAndSubscribe(this._xterm.raw.onResize, () => this._editorWidget.layout({ width: this._xtermElement.clientWidth, height: this._xtermElement.clientHeight })));
```

Comment:
This layout call is done twice with the same argument, good thing to pull into a `_layout` function.